Current Version/fade.py

```diff
@@ -0,0 +1,23 @@
+import pygame
+
+pygame.display.init()
+screen = pygame.display.set_mode((600, 400))
+image1 = pygame.image.load("image1.png").convert()
+image2 = pygame.image.load("image2.png").convert()
+clock = pygame.time.Clock()
+
+# fade in the image1
+for i in range(255):
+    screen.fill((0, 0, 0))
+    image1.set_alpha(i)
+    screen.blit(image1, (0, 0))
+    pygame.display.flip()
+    clock.tick(60) #changes transition speed
+
+#fade transition
+for i in range(255):
+    screen.blit(image1, (0, 0))
+    image2.set_alpha(i)
+    screen.blit(image2, (0, 0))
+    pygame.display.flip()
+    clock.tick(60) #changes transition speed
```

Comment:
I presume that this code is going to be integrated elsewhere. Consider how you could make is a more general function using arguments.